test/all.js

```diff
@@ -69,6 +73,30 @@ test('checkout query param (hyperdrive)', async t => {
   )
 })
 
+test('can add a non-ready drive with key', async function (t) {
+  const store = new Corestore(RAM.reusable())
+  const drive = new Hyperdrive(store.namespace('drive'))
+  await drive.put('/file', 'here')
+  const key = drive.key
+  const zKey = z32.encode(key)
+
+  await drive.close()
+
+  const serve = tmpServe(t)
+  await serve.ready()
+  const reDrive = new Hyperdrive(store.namespace('drive'), key)
+
+  t.is(reDrive.opened, false)
+  serve.add(reDrive) // No error
+
+  // Sanity check: can get file
```

Comment:
No comments!

Reply:
Will remove, but I don't think these are bad comments, they help explain the flow/intention of the tests. Without the comments, it seems as if the check is on whether the file is available, but the actual check is on whether `serve.add(reDrive)` doesn't throw

Reply:
Ah true true! I was already used to disregard comments, but the comment should say that the drive is not ready but key is sync

Reply:
Saying "No error" is not descriptive enough for what is happening

Reply:
Added more descriptive comment